### ISSUE TITLE
Cover Generator file actions are not accessible to screen reader users

### ISSUE THREAD

**Issue by greatislander** [pressbooks/pressbooks#2927]
### Prerequisites

* [x] Can you reproduce the behavior you wish to address with [all other plugins deactivated](http://codex.wordpress.org/Multisite_Network_Administration#Plugins)?
* [x] Are you running the [latest version of Pressbooks](http://github.com/pressbooks/pressbooks/releases/latest/) and the [latest version of WordPress](http://codex.wordpress.org/Upgrading_WordPress)?
* [x] Are you experiencing this issue with one of our built-in root or book themes?
* [x] Did you [perform a cursory search](https://github.com/issues?q=+is%3Aissue+repo%3Apressbooks%2Fpressbooks) to see if your bug or enhancement is already listed?

### Description

The cover generator page displays icons for generated covers. The icons are wrapped in a link which (as of #2926) will have an accessible label for screen reader users. However, the only way to delete an individual cover is to hover over the icon, which reveals a delete button below the icon (as well as a redundant download link). This behaviour should be improved.

### Steps to Reproduce

1. Examine the download icons on the cover generator page.

**Expected behavior:** The secondary actions should be accessible to screen reader users and keyboard/touch device users (e.g. tablets) who do not have a pointer that they can hover over the icon. The redundant download link should probably be removed.

**Actual behavior:** Issues identified in the issue description.

### System Information

Pressbooks: dev-dev (as of https://github.com/pressbooks/pressbooks/commit/05870e713e54f91330abf1e1ce50a3dc4218a0c8)
